src/sentry/tasks/derive_code_mappings.py

```diff
@@ -103,14 +106,17 @@ def derive_code_mappings(
     if data["platform"].startswith("php") and not features.has(
         "organizations:derive-code-mappings-php", org
     ):
+        logger.info("Php automatic code mappings are not open access", extra=extra)
```

Comment:
It's very tempting to keep adding logs to our code (I've done so in the past as you can see in this code), however, we keep increasing our GCP logs cost following this pattern.

If you're using logging in GCP for debugging in production, you should not use that pattern but use metrics instead. If you need the logging for writing tests use `logger.debug()` so we don't litter the production logs.

Let me know if it's none of the cases and we can figure out together how to go about it.

Reply:
Removed in the most recent commit - thanks for the feedback @armenzg! This was indeed just for testing in production